operators/aqua/6.2.2/manifests/aqua-operator.v6.2.2.clusterserviceversion.yaml

```diff
@@ -11,10 +11,10 @@ metadata:
     # is compatible with it. Please, ensure that your project is no longer using these API(s) and that you start to
     # distribute solutions which is compatible with Openshift 4.9.
     # For further information, check the README of this repository.
-    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.8"}]'
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.9"}]'
```

Comment:
The comment on top is wrong and does not matches with the value `olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.9"}]'`

Why are you trying to set here `olm.maxOpenShiftVersion` = v4.9?
It would block cluster-admins upgrades from OCP 4.9 to 4.10.
We have no Kubernetes APIs which were removed on 1.23 and in the Openshift vendor 4.10

So, why do you want to block cluster admins' upgrades when they have this Operator installed?
Why this Operator cannot work on 4.10? 

Also, did you make this version compatible with 4.9? Is not this version using APIs that were removed in 4.9?
See that you can check it locally with the command `operator-sdk bundle validate ./bundle --select-optional suite=operatorframework --optional-values=k8s-version=1.22`

Could you please fix it?

Reply:
c/c @mvalarh @J0zi @yossig-aquasec